django/db/models/sql/query.py

```diff
@@ -2338,6 +2343,34 @@ def add_ordering(self, *ordering):
         else:
             self.default_ordering = False
 
+    @property
+    def orderby_issubset_groupby(self):
+        """
+        Return true if order_by of query is subset of group_by.
+        """
+        if self.group_by is False:
+            # there is no use case for that, but we need to be sure
+            return False
+        if self.group_by in (None, True):
+            # it seems like there is no aggregation at all (None)
+            # or there are all required groupbies(True) generated automatically
+            # from models fields - so, it is safe to clear ordering
```

Comment:
Minor but the _it is safe to clear ordering_ assumes the sole caller will always be for order clearing purpose.

Maybe something along the lines of 

```
When grouping by all fields the order by is necessarily a subset of the group by.
```